evaluate_library_access_from_output_tsv/read_and_merge_library_access_datasets.R

```diff
@@ -0,0 +1,43 @@
+#### Read the datasets ####
+
+read_and_merge_library_access_datasets <- function(
+  lzma_compressed_library_access_tsv_location,
+  original_dataset_with_oa_color_column_location
+){
+  # '<<-' at several places below will add the variable into our global scope,
+  # beyond this function.
+  lzma_compressed_library_access_tsv <<- readr::read_tsv(
+    lzma_compressed_library_access_tsv_location
+  )
+  # View(lzma_compressed_library_access_tsv)  # Check the dataset
+  
+  # Create a temporary filepath for downloading the original dataset.
+  # Then download and read it.
+  tmp_filpath_for_original_dataset <- tempfile()
```

Comment:
This is all unnecessary I believe... readr should be able to read from the URL

Reply:
Progress update: I'm switching this to

```
original_dataset_with_oa_color_column <<- readr::read_tsv(
    original_dataset_with_oa_color_column_location
  )
```

but am now getting an error `Error in make.names(x) : invalid multibyte string at '<fd>7zXZ'`, which I'm now working to address. I've not worked with `readr` and compressed files, though, and so thought to ask first, if you see this soon: Have you seen that error before in this context? If so, do you remember how you addressed it?

Reply:
Googling the error brought up [this page](https://qiita.com/uri/items/ad2ce23ec5476b17f0c4) in Chinese. Translated potion relevant to error is:

> It suddenly became an error. `read_csv()` Assumes UTF-8 as the default file encoding. Apparently this file seems not to have been saved in UTF - 8. Now you can specify `read_csv()` the value of the file encoding `locale` using arguments , so let's deal with that.

`original_dataset_with_oa_color_column_location` seems to refer to the [file here](https://github.com/greenelab/scihub/blob/4172526ac7433357b31790578ad6f59948b6db26/data/state-of-oa-dois.tsv.xz), which should be UTF-8 encoded, so not sure. Try a different locale maybe?

Reply:
Yeah, that was my thought, too. No luck yet, but I'll keep at it (playing with different locales)!